cmake/projects/tiny-process-library/hunter.cmake

```diff
@@ -11,9 +11,9 @@ hunter_add_version(
     VERSION
     2.0.2
     URL
-    "https://gitlab.com/eidheim/tiny-process-library/-/archive/v2.0.2/tiny-process-library-v2.0.2.tar.gz"
+    "https://github.com/Bjoe/tiny-process-library/archive/624c2b61a32cb1544ec34f73bf1c5d87a8e2f19e.zip"
```

Comment:
Need merge of PR https://github.com/cpp-pm/tiny-process-library/pull/1

Reply:
@Bjoe Merged and released: https://github.com/cpp-pm/tiny-process-library/releases/tag/v2.0.2-p0

Reply:
Done